packages/fuels-accounts/src/kms/aws/client.rs

```diff
@@ -0,0 +1,62 @@
+use aws_config::{
+    default_provider::credentials::DefaultCredentialsChain, BehaviorVersion, Region, SdkConfig,
+};
+
+#[derive(Debug, Clone)]
+pub struct AwsConfig {
+    sdk_config: SdkConfig,
+}
+
+impl AwsConfig {
+    pub async fn from_environment() -> Self {
+        let loader = aws_config::defaults(BehaviorVersion::latest())
+            .credentials_provider(DefaultCredentialsChain::builder().build().await);
+
+        Self {
+            sdk_config: loader.load().await,
+        }
+    }
+
+    #[cfg(feature = "test-helpers")]
+    pub async fn for_local_testing(endpoint_url: String) -> Self {
+        let sdk_config = aws_config::defaults(BehaviorVersion::latest())
+            .credentials_provider(aws_sdk_kms::config::Credentials::new(
+                "test",
+                "test",
+                None,
+                None,
+                "Static Test Credentials",
+            ))
+            .endpoint_url(endpoint_url)
+            .region(Region::new("us-east-1")) // Test region
+            .load()
+            .await;
+
+        Self { sdk_config }
+    }
```

Comment:
Not everybody is going to have their credentials set to `test` and `test`. 

We should expose the credentials as arguments as well.

Maybe rename it as `for_testing` because it doesn't have to be local.